Source/WebCore/layout/formattingContexts/inline/InlineFormattingContext.cpp

```diff
@@ -158,11 +158,11 @@ InlineLayoutResult InlineFormattingContext::layout(const ConstraintsForInlineCon
     }
 
     if (TextOnlySimpleLineBuilder::isEligibleForSimplifiedTextOnlyInlineLayoutByContent(inlineContentCache().inlineItems(), layoutState().placedFloats()) && TextOnlySimpleLineBuilder::isEligibleForSimplifiedInlineLayoutByStyle(root().style())) {
-        auto simplifiedLineBuilder = TextOnlySimpleLineBuilder { *this, root(), constraints.horizontal(), inlineItemList };
+        auto simplifiedLineBuilder = TextOnlySimpleLineBuilder { *this, root(), constraints.horizontal(), inlineItemList, inlineContentCache().textSpacingContext() };
```

Comment:
(text-only) _**simple**_ line builder is supposed to stay simple so that it can process content in a performant manner. -and it's important to keep it that way (isEligibleForSimplifiedTextOnlyInlineLayoutByContent should probably return false for autospacing) -same for range builder as it is just a wrapper around the simple text only builder.